### PR TITLE
koord-descheduler: init evictor plugin first

### PR DESCRIPTION
**Pull request by eahydra**
Signed-off-by: Joseph <joseph.t.lee@outlook.com>

### Ⅰ. Describe what this PR does

Other plugins may depend on the evictor plugin, so we should initialize the evictor plugin first.

### V. Checklist

- [x] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [x] All checks passed in `make test`

### CODE REVIEW

**Comment by jasonliu747** [comment]
why do we need this change? add a brief summary in PR description 😉

**Comment by eahydra** [comment]
> why do we need this change? add a brief summary in PR description 😉

There is a comment in code "Other plugins may depend on the evictor plugin, so we should initialize the evictor plugin first."

File: pkg/descheduler/framework/runtime/framework.go [hunk 1]
```diff
@@ -219,32 +244,33 @@ type extensionPoint struct {
...
 	}
 }
 
-func (f *frameworkImpl) pluginsNeeded(plugins *deschedulerconfig.Plugins) sets.String {
-	pgSet := sets.String{}
-
+func (f *frameworkImpl) getOtherExtensionPoints(plugins *deschedulerconfig.Plugins) []extensionPoint {
 	if plugins == nil {
-		return pgSet
+		return []extensionPoint{}
```
Comment (saintube) [inline pkg/descheduler/framework/runtime/framework.go:259]: how about `return nil`

**Pull request merged**